Docs: Add MD for risky-octal rule (#2574)
ansible/ansible-lint: src/ansiblelint/rules/risky_octal.md

# risky-octal

> This rule checks that Octal/Numeric file permissions must contain leading zero or be a string.

Reviewer: ```suggestion
This rule checks that octal file permissions either contain a leading zero or are a string value.
```
